ci/tests/general/general_vdisk.py

```diff
@@ -222,3 +233,31 @@ def get_filesystem_location(vpool, vdisk_name):
         else:
             raise RuntimeError('Invalid hypervisor type specified: {0}'.format(hv_type))
         return location
+
+    @staticmethod
+    def get_config_params(vdisk):
+        """
+        :param vdisk: vdisk to retrieve config params from
+        :return: {} containing config params
+        """
+        status, params = GeneralVDisk.api.execute_get_action('vdisks', vdisk.guid, 'get_config_params', wait=True)
+        assert status is True,\
+            'Retrieving config params failed: {0} for vdisk: {1} - {2}'.format(status, vdisk.name, params)
+
+        assert 'metadata_cache_size' in params,\
```

Comment:
In function callers we assume that 'write_buffer' is also part of the params, so why not check for presence of all key-value pairs?